app/controllers/events_controller.rb

```diff
@@ -7,7 +7,7 @@ class EventsController < ApplicationController
 
   # GET /events
   def index
-    @events = Event.sorted_by_start_date(!can?(:edit, Event)).reverse
+    @events = Event.sorted_by_start_date(!can?(:view_unpublished, Event))
```

Comment:
Yes I think we'll tackle ordering as part of #304 once this is merged. Original intent was, since the main page starts in the present, to show the next event (so ascending order) and on the list page use descending order, since we don't want to start looking at all the past events.